cloudferrylib/os/actions/check_quotas.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from novaclient import exceptions as nova_exceptions
+
+from cloudferrylib.base import exception as cf_exceptions
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import log
+from cloudferrylib.utils import proxy_client
+from cloudferrylib.utils import utils
+
+
+LOG = log.getLogger(__name__)
+
+
+class CheckQuotas(action.Action):
+    """Verify source and destination clouds support quotas.
+    If not limited and keep usages quotas -
+    do not start migration and fail with error message.
+    """
+
+    def run(self, **kwargs):
+        LOG.info("Checking quotas support for SRC and DST clouds...")
+        check_quotas(self.src_cloud)
+        check_quotas(self.dst_cloud)
+        LOG.info("SRC and DST clouds support quotas (Nova "
+                 "quotas) API. Migration can proceed.")
+
+
+def check_quotas(cloud):
```

Comment:
Move this function into the class, i don't see any reasons, why it should be a separate. Please write the code in same style as other code of project.